spec/support/utilities_spec.rb

```diff
@@ -1,3 +1,16 @@
-require 'support/sign_in_helper_spec'
```

Comment:
one problem though...We still get the Capybara::ElementNotFound:
error...tried to include the sign_in_helper but capybara still expects a
button.

On 1 September 2015 at 18:43, techylite <notifications@github.com> wrote:

> In spec/support/utilities_spec.rb
> <https://github.com/rubymonsters/speakerinnen_liste/pull/392#discussion_r38434497>
> :
>
> > @@ -1,3 +1,16 @@
> > -require 'support/sign_in_helper_spec'
>
> Will do . That seems to help with a problem we hard.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rubymonsters/speakerinnen_liste/pull/392/files#r38434497>
> .
>



-- 
Esther Monchari
Business Information Technology
Skype Esther.Monchari